### ISSUE TITLE
Allow search plugin to return multiple fields

### ISSUE THREAD

**Issue by karzak** [DMS-Aus/Roam#380]
I have a layer (Buffer), with two columns of interest (address, status). Users want to be able to search for all buffers where status = 'In Progress'. However, the way the search plugin works, this just returns a list of buffers, with no address to distinguish one from the other. Being able to return both the status and the address would be a big help.

![roam_searchplugin_issue](https://user-images.githubusercontent.com/10601061/27914189-67a7d6d0-6230-11e7-9cc8-2b94c8732078.PNG)

**Comment by bzeeb** [comment]
Good idea however no time to implement at the moment, all of the search code is self contained within the search_plugin folder if you would like to take a shot at it.

You would need to add a new config field defining the extra info to display (column header), pack this in to the search index then extract around lines 220 - 230 in search.py
